client/src/game/GameLoop.js

```diff
@@ -53,5 +56,23 @@ export const gameLoop = (canvas) => {
       player.direction = DIRECTIONS.FRONT;
     }
   }
+
+  for (let index = 0; index < plattforms.length; index++) {
+    let platform = plattforms[index];
+    if (
+      player.top > platform.top - player.height &&
+      player.oldTop < platform.top - 0.9 * player.height &&
+      player.left > platform.left - 0.5 * player.width &&
+      player.left < platform.left + platform.width - 0.5 * player.width
```

Comment:
What do the values 0.9 or 0.5 stand for? Is it possible to define these in a variable if you are using them in different places? It would also make it easier to read and understand the code.

Reply:
It's a bit confusing, that's true. I opened a new pullRequest from this one with Leon and going to do this :D